Snabble/UI/Utilities/AlertStyle.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  AlertStyle.swift
+//
+//  Copyright © 2021 snabble. All rights reserved.
+//
+
+import SDCAlertView
+
+public final class SnabbleAlertStyle: AlertVisualStyle {
```

Comment:
Wollen wir die Subclass öffentlich bereitstellen? Ich würde es bei den vordefinierten Styles belassen.

Reply:
Apps sollen in der Lage sein, Alerts zu zeigen die genauso aussehen wie unsere, wir brauchen das in snabble auch an drei oder vier Stellen.